projects/plugins/jetpack/modules/search/instant-search/lib/test/api.test.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * @jest-environment jsdom
+ */
+/**
+ * Internal dependencies
+ */
+import { generateDateRangeFilter } from '../api';
+
+describe( 'generateDateRangeFilter', () => {
+	test( 'generates correct ranges for yearly date ranges', () => {
+		expect( generateDateRangeFilter( 'something', '2020-01-01 00:00:00', 'year' ) ).toEqual( {
+			range: { [ 'something' ]: { gte: '2020-01-01', lt: '2021-01-01' } },
+		} );
+	} );
+	test( 'generates correct ranges for monthly date ranges', () => {
```

Comment:
Would it be good if we cover month 9 / September for the coverage of line 93 in api.js ? 😄